test/release/examples/primers/taskParallel.chpl

```diff
@@ -74,31 +80,38 @@ writeln("3: output from main task");
 
 Coforall Loops
 --------------
-Another more structured form of task parallelism is the
-``coforall`` loop.  This loop form is like a ``for`` loop in which
+Another structured form of task parallelism is the
+``coforall`` loop.  This loop form is like a ``for`` loop, except that
 each iteration of the loop is executed by a distinct task.  Similar
-to the ``cobegin`` statement, the main thread of execution does not
+to the ``cobegin`` statement, the original task does not
 continue until the tasks created for each iteration have completed.
 */
 writeln("4: ### The coforall loop ###");
 
+config const n = 10;
+
 coforall i in 1..n {
-  writeln("4: output from spawned task 1 (iteration ", i, ")");
-  writeln("4: output from spawned task 2 (iteration ", i, ")");
+  writeln("4: output 1 from spawned task ", i);
+  writeln("4: output 2 from spawned task ", i);
 }
 
-// While the order of output within an iteration is deterministic (``1``
-// executes before ``2``), the order of output relative to other
-// iterations is not defined.  As with the ``cobegin`` statement, the output
-// from within the ``coforall`` loop will always precede the following
-// output.
+// While the statements within the loop body will execute in the
+// normal way (so in this case, the first ``writeln()`` will execute
+// before the second), the order of execution relative to other
+// iterations is undefined.
+
+// As with the ``cobegin`` statement, the original task will wait
+// until the ``coforall``'s child tasks have completed before
+// proceeding.  As a result, in this example, all output from within
```

Comment:
Two sentences starting with "As" in a row, maybe swap for "This means" or something?